src/lib/operators/table_scan/is_null_table_scan_impl.cpp

```diff
@@ -13,84 +13,77 @@
 
 namespace opossum {
 
-IsNullTableScanImpl::IsNullTableScanImpl(const std::shared_ptr<const Table>& in_table, const ColumnID base_column_id,
+IsNullTableScanImpl::IsNullTableScanImpl(const std::shared_ptr<const Table>& in_table, const ColumnID column_id,
                                          const PredicateCondition& predicate_condition)
-    : BaseSingleColumnTableScanImpl{in_table, base_column_id, predicate_condition} {
+    : AbstractSingleColumnTableScanImpl{in_table, column_id, predicate_condition} {
```

Comment:
Your suggestion is to inherit from `AbstractTableScanImpl`? Makes sense, just want to make sure that I get you right.

Reply:
I think that makes more sense

Reply:
Done. I guess we should rename `AbstractSingleColumnTableScanImpl` then, because it is no more the super class of *all* single column scans. I can't come up with a good name though.

Reply:
I know what you mean, but I cannot come up with a better name either. Not totally clean, but I would leave it as it is, probably